src/skin/qml/qmllibraryproxy.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+#include <QObject>
+
+class Library;
+
+namespace mixxx {
+namespace skin {
+namespace qml {
+
+class QmlLibraryTrackListModel;
+
+class QmlLibraryProxy : public QObject {
+    Q_OBJECT
+    Q_PROPERTY(mixxx::skin::qml::QmlLibraryTrackListModel* model MEMBER m_pModel CONSTANT)
```

Comment:
from https://doc.qt.io/qt-5/properties.html:
> Here is an example showing how to export member variables as Qt properties using the MEMBER keyword. Note that a NOTIFY signal must be specified to allow QML property bindings.

I guess that is untrue when the property is also marked CONSTANT? since
> The presence of the CONSTANT attribute indicates that the property value is constant. For a given object instance, the READ method of a constant property must return the same value every time it is called. This constant value may be different for different instances of the object. A constant property cannot have a WRITE method or a NOTIFY signal.

Reply:
Yes, I think so. It works.